### PR TITLE
Minor typo fixes

### PR DESCRIPTION
**Pull request by EfremBraun**

### CODE REVIEW

**Comment by EfremBraun** [comment]
Er, give me a few more minutes. Deadline is???

**Comment by davidlmobley** [comment]
Sorry, I thought you were done so I just fixed a conflict in your branch (very minor).

**Comment by davidlmobley** [comment]
I'd ideally like to submit before 9 am because I have back-to-back stuff beginning at 9 am and I promised it "first thing" today. :)

**Comment by EfremBraun** [comment]
Finished! Pull when you'd like.

File: paper/basic_training.tex [hunk 1]
```diff
@@ -310,6 +310,10 @@ \subsubsection{Key concepts}
 In this type of non-equilibrium situation, the distribution of configurations will not follow a Boltzmann factor distribution.  
 Specialized simulation approaches are available to study such systems~\cite{Chong:2017:CurrentOpinioninStructuralBiology,  Zuckerman:2017:AnnuRevBiophys} but they are not beginner-friendly.  
 Non-equilibrium molecular concepts pertinent to cell biology have been discussed at an introductory level (e.g. \url{http://www.physicallensonthecell.org/}).
+Notably, many experiments are conducted at non-equilibrium conditions; for example, membrane diffusion coefficients are commonly measured by setting up a concentration gradient across the membrane and measuring the flux.
+It can be tempting to the beginner to setup an MD simulation in the same manner as such an experiment.
```
Comment (davidlmobley) [inline paper/basic_training.tex]: Note to self, "setup" should be "set up".

File: paper/basic_training.tex [hunk 2]
```diff
@@ -310,6 +310,10 @@ \subsubsection{Key concepts}
...
+However, maintaining non-equilibrium conditions is typically more complicated in an MD simulation than in an experiment as large reservoirs are commonly required.
+Frequently there exist equilibrium methods that can be used to obtain the same or similar information as a non-equilibrium experiment; users should seek to obtain familiarity with such methods before choosing to conduct a non-equilibrium MD simulation.
```
Comment (davidlmobley) [inline paper/basic_training.tex]: I tend to not like the "there exist" phrasing; it's more direct to just be direct, e.g. "Frequently, equilibrium methods can provide ..."

Comment (davidlmobley) [reply paper/basic_training.tex]: (I can fix this after merging this.)

**Pull request merged**